app/scripts/move-popup/dimMovePopup.directive.js

```diff
@@ -45,9 +45,9 @@
     };
   }
 
-  MovePopupController.$inject = ['$rootScope', 'dimStoreService', 'dimItemService', 'ngDialog', '$q', 'toaster'];
+  MovePopupController.$inject = ['$scope', '$rootScope', 'dimStoreService', 'dimItemService', 'ngDialog', '$q', 'toaster'];
```

Comment:
any reason for this addition? don't seem to be using it anywhere else in this directive, maybe we could consider removing it?